frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -60,24 +64,36 @@ const DatasetRow: FC<Props> = ({
 }) => {
   const queryCache = useQueryCache();
 
-  const queryResult = useDatasetStatus(
+  const datasetStatusResult = useDatasetStatus(
```

Comment:
changed variable name to `datasetStatusResult`, since we have multiple queries in this component and `queryResult` could be too generic